molgenis-app-manager/src/main/resources/templates/view-app.ftl

```diff
@@ -29,4 +29,7 @@
 
 <#if app.includeMenuAndFooter>
     <@footer version/>
-</#if>
\ No newline at end of file
+    <script>
+      window.molgenisFooter.appVersion = '${app.version}'
+    </script>
+</#if>
```

Comment:
This makes the resulting html a bit wonky:
```
    <div id="molgenis-footer"></div>
    <script type=text/javascript>
        window.molgenisFooter = {
                version: '0.0.0-SNAPSHOT-PR-8099-2',
                buildDate: '2019-01-02 08:32 UTC'}
    </script>
    <script type=text/javascript src="/js/bootstrap-4/footer/molgenis-footer.js?RRg4bQ"></script>
</body>
</html>
    <script>
      window.molgenisFooter.appVersion = '1.0.4'
    </script>
```

Have you considered adding it as an optional parameter to the footer macro?